packages/components/psammead-brand/src/index.jsx

```diff
@@ -231,7 +231,7 @@ const Brand = props => {
           <StyledBrand {...props} />
         )}
         {skipLink}
-        {scriptLink}
+        {scriptLink && <div>{scriptLink}</div>}
```

Comment:
We should add snapshot tests that cover this

Reply:
This is already covered https://github.com/bbc/psammead/blob/4187ccd1d94d675d7425a74d0812aabe231fbe09/packages/components/psammead-brand/src/index.test.jsx#L24-L79

Reply:
Oh, I see, this covers it. https://github.com/bbc/psammead/blob/4187ccd1d94d675d7425a74d0812aabe231fbe09/packages/components/psammead-brand/src/index.test.jsx#L138-L191